shared/src/business/entities/DocketEntry.ts

```diff
@@ -185,7 +188,11 @@ export class DocketEntry extends JoiValidationEntity {
   public signedJudgeUserId?: string;
   public strickenBy?: string;
   public strickenByUserId?: string;
-  public workItem?: any;
+
+  // These are optional fields set for the UI
+  public qcComplete?: boolean;
+  public qcViewed?: boolean;
+  public workItemId?: string;
```

Comment:
I would love to just remove these fields from `DocketEntry.ts` altogether ... but `genericHandler` makes this difficult.